docs/quick_start.md

```diff
@@ -143,28 +318,37 @@ Summary of differences against `dev`:
 │   └── sqlmesh_example.incremental_model
 └── Indirectly Modified:
     └── sqlmesh_example.full_model
----                                                                                                             
-                                                                                                                
-+++                                                                                                             
-                                                                                                                
-@@ -1,6 +1,7 @@                                                                                                 
-                                                                                                                
- SELECT                                                                                                         
-   id,                                                                                                          
-   item_id,                                                                                                     
-+  'z' AS new_column,                                                                                           
-   ds                                                                                                           
- FROM sqlmesh_example.seed_model                                                                                
- WHERE                                                                                                          
+--- 
+
++++ 
+
+@@ -1,6 +1,7 @@
+
+ SELECT
+   id,
+   item_id,
++  'z' AS new_column,
+   ds
+ FROM sqlmesh_example.seed_model
+ WHERE
 Directly Modified: sqlmesh_example.incremental_model (Non-breaking)
 └── Indirectly Modified Children:
     └── sqlmesh_example.full_model
 Models needing backfill (missing dates):
-└── sqlmesh_example__dev.incremental_model: (2020-01-01, 2023-04-27)
-Enter the backfill start date (eg. '1 year', '2020-01-01') or blank for the beginning of history: 
+└── sqlmesh_example__dev.incremental_model: 2020-01-01 - 2023-05-31
+Enter the backfill start date (eg. '1 year', '2020-01-01') or blank to backfill from the beginning of history:
+```
+
+Lines 5-9 of the output summarize the differences between the modified pipeline and the existing `dev` environment, detecting that we directly modified `incremental_model` and that `full_model` was indirectly modified because it selects from the incremental model. 
+
+On line 23, we see that SQLMesh semantically understood that the change was additive (added a column not used by `full_model`) and was automatically classified as a non-breaking change.
```

Comment:
I'm not sure if "semantically understood" is a valid phrase here.

An alternative to consider: "[..] the change was _semantically additive_ [..]"